asv_bench/benchmarks/indexing.py

```diff
@@ -546,12 +546,9 @@ def time_chained_indexing(self, mode):
 
 
 class Block:
-    params = [
-        (True, "True"),
-        (np.array(True), "np.array(True)"),
-    ]
+    params = [True]
```

Comment:
`"True"` would now raise, so I'm removing it

Reply:
Could you just hardcode `True` in the only benchmark where its used?